Readme.md

```diff
@@ -582,6 +583,21 @@ next((Test Class, 13))
 completed
 ```
 
+#### count
+
+The `count` operator counts the number of items emitted by an Observable. If predicate exists, then counts the number of items satisfying it.
```

Comment:
Please change this to : 

> Emit the number of items emitted by an Observable once it terminates with no errors. If a predicate is given, only elements matching the predicate will be counted. 

Also, reference the official ReactiveX page: http://reactivex.io/documentation/operators/count.html